internal/service/searchindex/resource_search_index_migration_test.go

```diff
@@ -4,68 +4,15 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
-	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
 )
 
 func TestMigSearchIndex_basic(t *testing.T) {
-	var (
-		clusterInfo  = acc.GetClusterInfo(t, nil)
-		indexName    = acc.RandomName()
-		databaseName = acc.RandomName()
-		config       = configBasic(clusterInfo.ProjectIDStr, indexName, databaseName, clusterInfo.ClusterNameStr, clusterInfo.ClusterTerraformStr, false)
-	)
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { mig.PreCheckBasic(t) },
-		CheckDestroy: acc.CheckDestroySearchIndex,
-		Steps: []resource.TestStep{
-			{
-				Config:            config,
-				ExternalProviders: mig.ExternalProviders(),
-				Check: resource.ComposeTestCheckFunc(
-					checkExists(resourceName),
-					resource.TestCheckResourceAttr(resourceName, "name", indexName),
-					resource.TestCheckResourceAttrSet(resourceName, "project_id"),
-					resource.TestCheckResourceAttr(resourceName, "cluster_name", clusterInfo.ClusterName),
-					resource.TestCheckResourceAttr(resourceName, "database", databaseName),
-					resource.TestCheckResourceAttr(resourceName, "collection_name", collectionName),
-					resource.TestCheckResourceAttr(resourceName, "search_analyzer", searchAnalyzer),
-					resource.TestCheckResourceAttr(resourceName, "type", ""),
-				),
-			},
-			mig.TestStepCheckEmptyPlan(config),
-		},
-	})
+	testCase := mig.ConvertToMigrationTest(t, basicTestCase(t))
```

Comment:
there is duplication of mig in the name, what about something like mig.CreateTest?    also as it receives t in the param, it could just be:

mig.CreateTest(t, basicTestCase(t)) instead of 2 lines

Reply:
Thanks. Updated ✅  in 78a605e

Reply:
*And in [1b76bef](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2081/commits/1b76bef3dca52c9b06bf98f8f497d516b89db669)